packages/analytics/analytics-chart/src/components/CsvExportModal.vue

```diff
@@ -97,7 +97,6 @@ const { i18n } = composables.useI18n()
 const props = withDefaults(defineProps<{
   filename: string,
   modalDescription?: string,
-  selectedRange: string,
```

Comment:
This prop is not necessary since this can be computed from the chart data using the composable

Reply:
Nice!

It looks like you should update the corresponding component test to stop passing the `selectedRange` prop.  Otherwise, code LGTM; checking the sandbox.

Reply:
done